tests/utils/test_file_utils.py

```diff
@@ -90,6 +98,7 @@ def test_render_and_merge_yaml(tmpdir):
         "test_1": [1, 2, 3],
         "test_2": ["a", 1.2],
         "test_3": {"a": 2},
+        "test_4": extra_config,
```

Comment:
```suggestion
        "test_4": 123,
```

If we make the suggested change to the test to read `key`